modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-operator-supported_{context}"]
+= {oadp-full} support
+
+.Supported versions of {oadp-short}
+[width="100%",cols="20%,20%,20%,20%,20%, options="header",]
+|===
+
+|Version
+|{OCP-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+
+|1.4
+|4.14, 4.15, 4.16
+|10 Jul 2024
+|Release of 1.5
+|Release of 1.6
```

Comment:
OK.. same thing here: refer to https://access.redhat.com/support/policy/updates/openshift

* Extended update support ends: June 27, 2026	
* Extended update support Term 2 ends: June 27, 2027

Reply:
extended update support Term1 and Term 2 must be on OCP 4.16